app/models/space.rb

```diff
@@ -5,4 +5,7 @@ class Space
 
   property :id, Serial
   property :name, String
+  property :description, String
+  property :price, Float
```

Comment:
add validation for price to be 2dp

http://datamapper.org/docs/validations.html